pipelines/broad/dna_seq/germline/variant_calling/VariantCalling.changelog.md

```diff
@@ -1,5 +1,12 @@
+# 2.0.0
+2021-11-01
+
+* Added an optional step to reblock gVCFs, this step is included by default
+    * The WholeGenomeGermlineSingleSample and ExomeGermlineSingleSample pipelines now output reblocked gVCFs by default. To skip reblocking, add '"WholeGenomeGermlineSingleSample.BamToGvcf.skip_reblocking": true' or '"ExomeGermlineSingleSample.BamToGvcf.skip_reblocking": true' to the appropriate inputs
```

Comment:
Do we assume that people may want to run variant calling as a separate pipeline? In that case does variant calling now include reblock or is that downstream?

Reply:
Variant calling includes reblocking.  All (germline) roads lead to reblocking unless the user opts out by changing the inputs.  Liz raised the good point that DRAGEN will also be reblocked, which is what we're doing for AoU, but if this is supposed to be the functionally equivalent GATK-DRAGEN pipeline by default then I can turn that off.

Reply:
I believe @michaelgatzen said previously it was ok to have reblocking turned on for dragen-gatk modes.

Reply:
With the current definition of functional equivalence it won't have an effect on it, as we only base the evaluation on variant calls, not reference blocks (or any annotations)